views/components/Sidebar/SidebarFooter.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import {
+  Typography,
+  Card,
+  Avatar,
+  CardHeader,
+  CardContent,
+  CardActions,
+  Button,
+} from '@mui/material'
+import { Link } from 'react-router-dom'
+import { routes } from '../../routes/routes'
+import { FontSize14InRem } from '../../../constants'
+import './SidebarFooter.css'
+
+const SidebarFooter = ({ user, onLogout }) => {
+  return (
+    <Card
+      sx={{ border: 0, boxShadow: 0, color: 'text.primary', height: '236px' }}
+    >
+      <CardHeader
+        avatar={<Avatar alt={user.display_name[0]} src={user.icon} />}
+      />
+      <CardContent sx={{ pt: 0 }}>
+        <Typography sx={{ fontWeight: 600 }} variant="h5">
+          {user.display_name}
+        </Typography>
+        <Typography
+          sx={{ fontSize: FontSize14InRem, color: 'grey.A200' }}
+          variant="subtitle1"
+        >
+          Title {user.title}
+        </Typography>
+      </CardContent>
+      <CardActions
+        sx={{ fontSize: FontSize14InRem }}
+        className="sidebar_footer_actions"
+      >
+        <Button
+          variant="outlined"
+          component={Link}
+          to={routes.userAccount}
+          sx={{ borderColor: 'text.primary', color: 'text.primary' }}
+        >
+          Edit Profile
+        </Button>
+        <Button
+          variant="text"
+          sx={{ color: 'text.primary' }}
+          onClick={() => onLogout()}
+        >
+          Log Out
+        </Button>
+      </CardActions>
+      <div className="sidebar_footer_version">
```

Comment:
it's probably worth using a consistent style convention, like BEM, for these class names. I find BEM plays well with react because the "block" can be the component name. In this case, a class name like `SidebarFooter__version` could work.

Reply:
I've written this using CBEM

Reply:
what is CBEM?

Reply:
container block element modifier, similar to the syntax you've provided with the difference being yours is joined

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)